src/EditorFeatures/CSharpTest/RefactoringHelpers/RefactoringHelpersTests.cs

```diff
@@ -726,6 +726,35 @@ class TestAttribute : Attribute { }
 }";
             await TestMissingAsync<PropertyDeclarationSyntax>(testText);
         }
+
+        [Fact]
+        [WorkItem(35525, "https://github.com/dotnet/roslyn/issues/35525")]
+        public async Task TestMultipleExtractions()
```

Comment:
fwiw, i would prefer separate tests.  it's such a pain to have tests where something fails, you fix it, and then it fails on the next thing it does.

Reply:
This one is supposed explicitely test that both extractions can work at the same time, not that the two individual work correctly. There're individual tests for that. 

While that means effective supplication of tests, I feel like this "combined" test conveys a different message. The same could be achieved via comments in the two tests but I don't trust comments.

Reply:
Essentially if this test fails, some other that tests only one thing will fail too. This is to explicitely convey the idea that given one location multiple nodes should be retrievable. 

I.e if the some other test fails it can help discover a potential issue in "the extraction logic is sound but it breaks on the first type/header". Which might not be as obvious from the one standalone test.